doc/Overview.md

```diff
@@ -19,37 +19,153 @@
 // <http://www.gnu.org/licenses/>.
 -->
 
-## Abstract
+\section OverviewIntro What is Pinocchio?
 
-We introduce Pinocchio, an open-source software framework that implements rigid body dynamics algorithms and their
-analytical derivatives. Pinocchio does not only include standard algorithms employed in robotics (e.g. forward and
-inverse dynamics) but provides additional features essential for the control, the planning and the simulation of
-robots. In this paper, we describe these features and detail the programming patterns and design which make Pinocchio
-efficient. We also offer a short tutorial for easy handling of the framework.
+Pinocchio is a library for efficiently computing the dynamics (and derivatives) of a robot model, and behing, any articulated rigid-body model you would like (avatar in a simulator, skeletal model for biomechanics).
+Pinocchio is one of the most efficient library for computing the dynamics of articulated bodies.
+It implements the classical algorithms following the methods described in Featherstone 2009 book (many thanks to him).
+It also introduce efficient variations of some of these algorithms, some new ones and algorithms to compute the derivatives of the main algorithms.
 
-## Installation
+Pinocchio is open-source, mostly written in C++ with Python bindings, and distributed under LGPL-v3 licence.
+Contributions are welcome.
+
+In this doc, we will find the usual description of the library functionalities, a quick tutorial to catch over the mathematics behind the implementation, a bunch of examples about how to implement classical applications (inverse kinematics, contact dynamics, collision detection, etc) and a set of practical exercices for beginners.
+
+\section OverviewInstall How to install Pinocchio?
+
+Pinocchio is best installed from APT packaging on Ubuntu 14.04 and 16.04, from our repository.
```

Comment:
I think this section should replicate what is written on the website.
Because there is also binairies available for OSX which is not mentioned.
We also support 18.04, Fedore, BSD and so on.

Reply:
I tried here to give a fair opinion about what are the typical system using pinocchio. I think we have 90% of the user/dev of Pinocchio in u 14 and 16. Hence I am very confident that Pinocchio "works" on these 2 configs. Are you similarly confident with eg OSX and BSD? I advise we find a wording that render this opinion.

Reply:
For OS X yes, I use it on my Macbook, and Justin also routinely works on OS X as far as I know. Can't tell anything about brew the brew installation though, never tried it for pinocchio.

Reply:
According to @nim65s, Pinocchio works correctly on ubuntu 18, and it also builds correctly on Fedora and BSD (don't know about the tests though). I will add a reference to ubuntu 18 and to OSX

Reply:
I included 18.04 and Mac OS X. For Ubuntu 14.04, 16.04 and 18.04 mentioned APT, for Mac Homebrew. I said that it can be installed from source for other systems.
I am thinking now that in between we could add a sentence like "Pinocchio should also work on Fedora, BSD [etc.], but this possibility is not fully tested". Feel free to add this sentence, including any OS you like